examples/secret.tf

```diff
@@ -13,10 +13,10 @@ module "secrets_manager" {
 
   secrets = {
     // the key "test-secret-01" is used to create kubernetes resource and must only contain lowercase alphanumeric characters, dots and dashes
-    "test-secret-01" = {
+    "test-secret-one" = {
```

Comment:
is this just to test different combinations?

Reply:
yes